src/Form.jsx

```diff
@@ -1542,6 +1546,15 @@ function Form({
                                             handleValueChange(value, servar.key)
                                         );
                                     }}
+                                    onChange={(e) => {
```

Comment:
what is this for?

Reply:
On each change event on `servar` field, the remaining input element's values are getting reset because of not setting a change event to update these controlled components. 

This will fix the issue.

Reply:
Removed this change as this issue is not happening.